dist/assets/7d4bb0ef.app.min.js

```diff
@@ -0,0 +1 @@
ri},a.prototype.activate=function(){return this.element.classList.remove("hide")},a.prototype.deactivate=function(){return this.element.classList.add("hide")},a.prototype.moveToCurrentFrame=function(){return this.currentFrame()||this.initializeFrames(),this.element.style.top=""+-1*this.currentFrame().topPosition+"px",this.element.style.left=""+-1*this.currentFrame().leftPosition+"px"},a.prototype.resetFrame=function(){return this.frameIndex=0,this.moveToCurrentFrame()},a}(),b["default"]=c}),define("multiSceneMovie/vastRequest",["multiSceneMovie/beacon","exports"],function(a,b){"use strict";var c,d=a["default"],e=function(a,b){return function(){return a.apply(b,arguments)}};c=function(){function a(a,b,c){this.url=a,this.eventEmitter=b,this.callback=c,this.setTrackers=e(this.setTrackers,this),this.loadWrapperAdIfNeeded=e(this.loadWrapperAdIfNeeded,this),this.clickThrough=void 0,this.ready=void 0,this.url&&this.run()}var b;return a.prototype.valid=function(){return this.url?!0:!1},a.prototype.run=function(){return this.requestHandler(this.loadWrapperAdIfNeeded)},a.prototype.loadWrapperAdIfNeeded=function(a,b){var c,e,f;if(e=a.querySelector("Wrapper VASTAdTagURI"),!e)return b(a);f=e.textContent,this.xhr||(this.xhr=new XMLHttpRequest),this.xhr.addEventListener("load",function(c){return function(){var e,f,g,h,i;for(b(c.xhr.responseXML),c.xhr.removeEventListener("load",arguments,!1),f=a.querySelectorAll("Impression"),i=[],g=0,h=f.length;h>g;g++)e=f[g],i.push(new d(e.textContent));return i}}(this),!1);try{return this.xhr.open("GET",f),this.xhr.send(null)}catch(g){c=g}},a.prototype.setTrackers=function(a){var c,d,e,f;if(a){for(d=[],Array.prototype.forEach.call(a.querySelectorAll("Linear VideoClicks ClickThrough"),function(a){return d.push({event:"clickThrough",url:a.textContent,type:"img"})}),Array.prototype.forEach.call(a.querySelectorAll("Linear TrackingEvents Tracking"),function(a){return d.push({event:a.getAttribute("event"),url:a.textContent,type:"img"})}),Array.prototype.forEach.call(a.querySelectorAll("Impression"),function(a){return d.push({event:"impression",url:a.textContent,type:"img"})}),e=0,f=d.length;f>e;e++)c=d[e],b(this.eventEmitter,c);return this.mediaUrl=a.querySelector("Linear MediaFile").textContent,this.callback?this.callback():void 0}},b=function(a,b){var c,e;if(a)return e={creativeView:"movie:screen:appeared",impression:"movie:started",firstQuartile:"movie:played:firstQuartile",midpoint:"movie:played:half",thirdQuartile:"movie:played:thirdQuartile",complete:"movie:finished",clickThrough:"movie:clickThrough"},c=e[b.event],c?a.listen(c,function(){return new d(b.url,b.type)}):void 0},a.prototype.requestHandler=function(a){var b;this.xhr||(this.xhr=new XMLHttpRequest),this.xhr.addEventListener("load",function(b){return function(){return a(b.xhr.responseXML,b.setTrackers),b.xhr.removeEventListener("load",arguments,!1),b.ready=!0}}(this),!1);try{return this.xhr.open("GET",this.url),this.xhr.send(null)}catch(c){b=c}},a}(),b["default"]=c});
```

Comment:
Line is too long.<br>'a' is already defined.<br>Possible strict violation.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>A constructor name should start with an uppercase letter.<br>Expected '{' and instead saw 'throw'.<br>Expected '{' and instead saw 'return'.<br>Too many errors. (100% scanned).